### ISSUE TITLE
Sponsor block chapters

### ISSUE THREAD

**Issue by venkat2305** [polymorphicshade/NewPipe#281]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/NewPipe/issues) or [closed](https://github.com/polymorphicshade/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/polymorphicshade/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Sponsorblock introduced chapers support. 

### Why do you want this feature?

It makes the video watching experience better and we can see the chapters (if contributed by the community) even if the video uploader doesn't add timestamps 

### Why ist the feature relevant to this fork?

Because it's a sponsor block feature

**Comment by Midou36O** [comment]
Would that require having an account that have paid for the feature or would it simply just work? (This is considered cracking, unless Ajayy explicitly permits it) 🤔

**Comment by venkat2305** [comment]
> 

don't know. chapters are enabled on mine. didn't have to pay anything. Is it a paid feature?

**Comment by Midou36O** [comment]
It is a paid feature, unless it recently went free, but i've yet to see that (my sb extension on librewolf seems to still require me to pay to be able to access it.)

**Comment by venkat2305** [comment]
can you check , if you can see any chapters on this [video](https://www.youtube.com/watch?v=EExSSotojVI&t=16848s)

**Comment by Midou36O** [comment]
Nothing shows up

**Comment by venkat2305** [comment]
could you check on chrome?

**Comment by Midou36O** [comment]
No, really, nothing, even the sponsorblock extension shows the lock icon
![image](https://user-images.githubusercontent.com/45198486/201490021-52e55171-f501-4f74-856e-d008c341d787.png)
(I just installed sponsorblock on a test clean chromium profile)

**Comment by venkat2305** [comment]
ok. @polymorphicshade if it's a paid featyre. there's no point in adding this

**Comment by Midou36O** [comment]
Not exactly, it can be a nice bonus for those who wish to use it and paid for it.

**Comment by polymorphicshade** [comment]
I agree with @Midou36O, though chapter support won't be available for a while.

**Comment by DraconicNEO** [comment]
> It is a paid feature, unless it recently went free, but i've yet have to see that (my sb extension on librewolf seems to still require me to pay to be able to access it.)

it's a paid feature but only for the extension, the API doesn't have a paywall so the developers of other clients are free to implement it if they choose.

**Comment by DraconicNEO** [comment]
> Would that require having an account that have paid for the feature or would it simply just work? (This is considered cracking, unless Ajayy explicitly permits it) 🤔

the API is open access so technically anyone can implement it if they want. The license check is done through the extension.

**Comment by Midou36O** [comment]
I guess we will need @Ajayyy to clarify some things :P

**Comment by ajayyy** [comment]
Yes, as explained in the API docs, you are allowed to use the API for free.
